### ISSUE TITLE
Finetuning Layout Analysis with Different Classes

### ISSUE THREAD

**Issue by bely66** [PaddlePaddle/PaddleOCR#8131]
Hi Everyone,
I'm trying to finetune the layout analysis model on my dataset which is relatively small compared to the benchmarked dataset.
So starting from scratch here doesn't make sense.

The issue I'm facing is I don't wanna train lists and figures for my dataset and here's the question

Would it be hard to get the pre-trained network fully except for the classification head? so that I have the knowledge without being forced to train for the same classes?

**Comment by bely66** [comment]
@tink2123  Would it be hard to do this fine-tuning?

**Comment by Tanmay98** [comment]
Hi, is there any update on this? I tried fine-training using 250 images but it is not giving good accuracy while inferencing.
